packages/react-devtools-extensions/webpack.config.js

```diff
@@ -49,9 +51,11 @@ module.exports = {
   },
   plugins: [
     new DefinePlugin({
-      __DEV__: false,
-      __PROFILE__: false,
+      __DEV__,
       __EXPERIMENTAL__: true,
+      __EXTENSION__: true,
```

Comment:
This didn't need to be passed in as `process.NODE_ENV.TARGET`. It's only true for one of the Webpack config files (this one).